### ISSUE TITLE
Not able to install anything on Tiny10 21H2

### ISSUE THREAD

**Issue by wanderling** [QuestYouCraft/Microsoft-Store-UnInstaller#4]
New install of Tiny10 21H2 from the Archive and given a proper key. Not able to install store with this executable. I understand if you are not willing to support Tiny10 but if you could point me in the right direction that would be a comfort.

**Comment by QuestYouCraft** [comment]
It would be good to know the technical details in more detail (for example, which version of Tiny10 or Windows 10 is used). However, I think you need to install [Visual C++ Redistributable Package](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist) (preferably all versions) if you haven't already installed it. It would also be nice to provide a screenshot of the error in MSUI.

**Comment by wanderling** [comment]
So... I tried about twenty different things after your app failed. I did eventually get the Store to run but it never connects. Oh well...

I am going to close this issue but if you feel like tinkering I will drop as many of the details as possible.

For background, the Tiny 10 I used is [22H2 Beta 2 x64](https://archive.org/details/tiny-10-NTDEV) from Archive.org. I then gave it the key that was used in the previous installation on this machine. Installed the proper drivers for this machine (NUC7i7BNH) and proceeded to try to get the Store working. My ultimate goal is to reinstall defender antivirus (Not available on the store but requires the store to get installed). Again thanks for following up and sorry I can't get you more details. Cheers!
